index.html

```diff
@@ -1292,12 +1284,12 @@
 </aside>
 
 <aside class="example">
-  A user visits a mapping website that they regularly visit and clicks a locator icon; although
+  A user loads a mapping website that they regularly visit and clicks a locator icon; although
```

Comment:
I liked "visits".

Reply:
I like it too, but it repeats "visit" at short enough an interval to trip. Maybe make it "that they use regularly"?

Reply:
Went with visits and use.